public/service_portal/v0.1.0/openapi.json

```diff
@@ -428,6 +428,49 @@
                 }
             }
         },
+        "/portfolio_items/{portfolio_item_id}/icon": {
+            "get": {
+                "tags": [
+                    "users",
+                    "admins"
+                ],
+                "summary": "Fetches the specified portfolio item's icon information, reaching out to topology if necessary",
+                "operationId": "fetchServiceOfferingIconFromPortfolioItem",
+                "description": "Fetch the specified portfolio item's icon information.\n",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/PortfolioItemID"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Portfolio Item Icon",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/ServiceOfferingIcon"
+                                    }
+                                }
+                            }
+                        }
+                    },
+                    "401": {
+                        "$ref": "#/components/responses/Unauthorized"
+                    },
+                    "403": {
+                        "$ref": "#/components/responses/Forbidden"
+                    },
+                    "404": {
+                        "description": "Portfolio Item Not found"
```

Comment:
For 404, There is an interesting conversation for either the missing icon or missing portfolio.  In theory they're both the same error. We may want to adjust the text including both the Portfolio Item or the Icon.

Reply:
@lindgrenj6 @syncrou 
The IPP for Service Portal talks about consistent error reporting, we might have to switch over to that at some point.
https://docs.google.com/document/d/1sEgro5-lXYJGsxOdxFOcyz7EHilrnpBVkHT_m2IXogk/edit#heading=h.gnm5ox3wgszo